docs/docs/adding-tags-and-categories-to-blog-posts.md

```diff
@@ -238,11 +226,12 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
 Some notes:
 
 - Our GraphQL query only looks for data we need to generate these pages. Anything else can be queried again later (and, if you notice, we do this above in the tags template for the post title).
-- While making the tag pages, note that we pass `tag` through in the `context`. This is the value that gets used in the `TagPage` query to limit our search to only posts tagged with the tag in the URL.
+- We have referenced two `allMarkdownRemark` fields in our query. To avoid naming collisions we must alias one of them. We alias both to make our code more human-readable.
```

Comment:
```suggestion
- We have referenced two `allMarkdownRemark` fields in our query. To avoid naming collisions we must [alias](/docs/graphql-reference/#aliasing) one of them. We alias both to make our code more human-readable.
```

Linking to this could help some people who haven't heard of GraphQL aliases 🙂